public/usage-examples/graphics/draw_pixel_on_window/draw-pixel-on-window-1-simple.cpp

```diff
@@ -0,0 +1,26 @@
+#include "splashkit.h"
+
+int main()
+{
+    // Open a window titled "Falling Snow" with a size of 800x600
+    open_window("Falling Snow", 800, 600);
+
+    // Loop until the user requests to quit
+    while (!quit_requested())
```

Comment:
This caused the program to go unresponsive. I suggest removing this while loop and increase the number of pixels.